.travis.yml

```diff
@@ -1,10 +1,26 @@
 language: node_js
 node_js:
-  - "0.10"
-  - "0.12"
   - "4"
```

Comment:
I'd like to move our support policy to latest LTS and latest stable. It'll probably still work on 5, but the extra latency in testing isn't worth it, IMO.